app/2.0/start/toolbox/set-up.md

```diff
@@ -98,4 +104,5 @@ a page to your app.
     href="create-a-page">Next step: Create a page</a>
 
 [toolbox]: /2.0/toolbox/
+[shared styles]: /1.0/docs/devguide/styling.html#style-modules
```

Comment:
I think it's referring to this line

`[shared styles]: /1.0/docs/devguide/styling.html#style-modules`

Removed it.